.github/workflows/ci.yml

```diff
@@ -26,8 +26,10 @@ jobs:
                 python-version: 3.8
 
         -   name: Install python dependencies
-            run:
+            run: |
                 pip install -e .[pre-commit,tests]
+                pip uninstall -y aiida-quantumespresso
+                pip install git+https://github.com/aiidateam/aiida-quantumespresso.git@develop
```

Comment:
Is this still necessary??
Same below at line 68

Reply:
Nope, will remove.